src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.NoSuchElementException;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
+
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
Since the BuefferedReader br has a very large scope (accessible everywhere main method), you should most likely name this a bit longer so that it is easier to modify or search for in the future.